tools/Makefile.unix

```diff
@@ -697,3 +704,8 @@ apps_distclean:
 ifneq ($(APPDIR),)
 	$(Q) $(MAKE) -C "$(TOPDIR)/$(APPDIR)" TOPDIR="$(TOPDIR)" distclean
 endif
+
+check_format:
+	$(call colorecho,'Checking NuttX formatting with nxstyle')
```

Comment:
There is a missing dependency here.  tools/nxstyle[.exe] is not build by default but must be explictly built.  Can you add

`nxstyle$(EXEEXT):
    $(MAKE) -C tools -f Makfile.host nxstyle$(EXEEXT)`

check_format:  nxstyle$(EXEEXT)
...

Reply:
Unresolved... I don't see the change.

Reply:
Unresolaved again:  I still don't see the change

Reply:
One is enough!